src/extension.ts

```diff
@@ -62,10 +63,13 @@ export async function activate(context: vscode.ExtensionContext) {
 		}
 	});
 
-	// set up our completion provider
+	// set up our completion providers
 	let markdown:vscode.DocumentSelector = { scheme: 'file', language: 'markdown', pattern: '**/**.didact.md' };
 	vscode.languages.registerCompletionItemProvider(markdown, new DidactUriCompletionItemProvider());
 
+	let asciidoc:vscode.DocumentSelector = { scheme: 'file', language: 'asciidoc', pattern: '**/**.didact.adoc' };
+	vscode.languages.registerCompletionItemProvider(asciidoc, new DidactUriCompletionItemProviderAsciiDoc());
+
```

Comment:
as we have "DidactUriCompletionItemProviderAsciiDoc" maybe worth renaming the "DidactUriCompletionItemProvider" to "DidactUriCompletionItemProviderMarkdown"

Reply:
Since I am extending the main DidactUriCompletionItemProvider class, it seems weirder to extend a Markdown class for AsciiDoc, but maybe that's just me?

Reply:
Talked it through a bit and reorganized the classes a bit